--- multi-term.el	2016-03-03 00:46:03.895222222 -0300
+++ multi-term.el.new	2016-03-03 00:45:52.879222317 -0300
@@ -638,7 +638,12 @@
           (setq shell-name (read-from-minibuffer "Run program: " shell-name)))
       ;; Make term, details to see function `make-term' in `term.el'.
       (if multi-term-program-switches
-          (make-term term-name shell-name nil multi-term-program-switches)
+	  (if (stringp multi-term-program-switches)
+	      ;; Not breaking old stuff using multi-term-program-switches as string
+	      (make-term term-name shell-name nil multi-term-program-switches)
+	    ;; Passing multi-term-program-switches as a list
+	    (apply 'make-term term-name shell-name nil multi-term-program-switches))
+
         (make-term term-name shell-name)))))
 
 
